src/main/resources/application.yaml

```diff
@@ -2,12 +2,15 @@ spring:
   jpa:
     show-sql: true
     hibernate:
-      ddl-auto: none
+      ddl-auto: update
   datasource:
     driver-class-name: org.postgresql.Driver
-    url: jdbc:postgresql://localhost:5432/internship123
-    password: "0311"
+    url: jdbc:postgresql://localhost:5432/internship
+    password: "mama071991"
     username: postgres
+    initialization-mode: always
+    data:
+      - 'classpath:import.sql'
```

Comment:
is it supported to plug in 2 files? (each for appropriate entity)

Reply:
File import.sql has test data which fill out two tables : form and internship

Reply:
I understand it. Is it possible to split this file and plug in to application.yaml separately?

Reply:
Yes, I checked and got two filled tables out. Should I leave two separate files?

Reply:
I suggest to split it. It will be more readable.

Reply:
done